wiki/osu!_wiki/en.md

```diff
@@ -33,3 +33,9 @@ Despite having the *wiki* suffix in its name, the osu! wiki doesn't fully satisf
 
 - Users are encouraged to improve existing articles and create new ones, but basic knowledge of [GitHub](https://github.com) is expected, which raises the threshold necessary for entry;
 - All changes to the osu! wiki are pre-moderated and undergo the review process, where a contributor is expected to collaborate with the wiki maintainers.
+
+## Licence
+
+The majority of content in the osu!wiki is licensed under [CC-BY-NC 4.0](https://github.com/ppy/osu-wiki/blob/master/LICENCE.md)—in general, the materials of the wiki may be used in a non-commercial manner (see the [quick summary](https://tldrlegal.com/license/creative-commons-attribution-noncommercial-4.0-international-(cc-by-nc-4.0))). Some content may be covered by different third party licences, which will then be cited local to it and override the global licence.
+
+The "osu!" or "ppy" branding is *exempt from this* and is protected by trademark law. For information on use of these terms, [contact us](mailto:contact@ppy.sh).
```

Comment:
exempt doesn't seem like the correct terminology. it's not really exempt, just has addition clauses. probably best to copy what's mentioned on the main readme if you want to mention that here?

Reply:
yeah